src/conv/k-g/region_list.h

```diff
@@ -32,24 +32,31 @@
 
 #include "bot.h"
 
+struct Region {
+    Bot                                bot;
+    std::map<std::string, std::string> attributes;
+};
 
 class RegionList {
 public:
     RegionList(void);
 
     Bot& addRegion(const std::string& name);
 
-    Bot& getBot(const std::string& name);
+    Bot&                                getBot(const std::string& name);

```

Comment:
Why so many spaces?

Reply:
because there was the function const std::map<std::string, std::string> getAttributes() under it and I wanted to kep them next to each other. then we deleted getAttributes() and I forgot to bring it back :)